schema/DoctrineMigrations/Version20200512093347.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Application\Migrations;
+
+use Ivoz\Core\Infrastructure\Persistence\Doctrine\LoggableMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20200512093347 extends LoggableMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE HuntGroups CHANGE ringAllTimeout ringAllTimeout SMALLINT DEFAULT NULL');
```

Comment:
What about adding here

`UPDATE HuntGroups SET ringAllTimeout=NULL WHERE strategy!='ringAll'`

and corresponding in down

`UPDATE HuntGroups SET ringAllTimeout=0 WHERE strategy!='ringAll'`

Reply:
done